### PR TITLE
docs: Add documentation for updating multiple parameters at once

### LINKED ISSUE #105: docs: Include guidance for updating multiple query params at once
### Describe the problem

Currently there is no guidance for updating multiple query parameters at once. If you aren't sure of exactly sure of how the library works this can lead to unintended behavior and errors.

For instance if I have a date range component that should update 3 parameters simultaneously:
```ts
{
  period?: string;
  from?: number;
  to?: number;
}
``` 

In a change function I may make the mistake of writing it like: 
```js
const store = queryParameters();

const change = (value) => {
  $store.period = value.period;
  $store.from = value.from;
  $store.to = value.to;
}
```

In this case I would call `goto()` 3 times on the page causing the following error to appear in the browser:
```
sveltekit 'client.js Uncaught (in promise) Error: navigation aborted'
```

This could easily be avoided by using the `update` function on the store:
```js
const store = queryParameters();

const change = (value) => {
  store.update(v => {
    return {
      ...v,
      period: value.period,
      start: value.start,
      end: value.end
    }
  })
}
```

### Describe the proposed solution

Add documentation in the README with guidance on how to handle `batch updating` query parameters like shown above.

### PR DESCRIPTION
**Pull request by ieedan**
Adds guidance in the README on how to handle updating multiple parameters at once.

- Fixes #105

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the README.md with clarifications on query parameters usage in SvelteKit, including a new section on updating multiple parameters.
	- Corrected a typo in the warning section for improved accuracy.
- **Style**
	- Made formatting adjustments in multiple files, including consistent indentation and line breaks for improved readability.
	- Added newlines at the end of files where necessary to adhere to coding standards.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,35 @@ Just like with the single parameter case you can just update the store and the U
 
 writing in the input will update the state and the URL at the same time.
 
+### Updating multiple parameters at once
+
+When you need to update multiple different parameters at once you will want to use the `update()` method to update the store to prevent multiple calls of `goto()`.
```
Comment (paoloricciuti) [inline README.md:220]: This only happens when using `queryParameters` tho...can you please specify that in the documentation?

**Review by paoloricciuti — COMMENTED** [review]
Thanks for the contribution...i've left a small comment. Also a lot of those fixes are just some prettier misconfiguration, can you revert those?

File: README.md (hunk 1 above)
Comment (ieedan) [reply README.md:220]: Would it not happen if you had multiple queryParams? Doesn't it call goto the same way?

**Comment by ieedan** [comment]
Yeah sorry I just installed and ran format at the end.

**Comment by ieedan** [comment]
I will just close this and reopen it to git rid of the extra changes since its easier

**Pull request closed without merging**